src/modules/ui.js

```diff
@@ -0,0 +1,89 @@
+import { ulContainer, taskInput } from './selectors';
+import ArrayTask from './class-methods';
+
+// constant variables
+const CHECK = 'fa-square-check';
+const UNCHECK = 'fa-square';
+const LINE_THROUGH = 'lineThrough';
+
+let edit = false;
+export const listofTasks = new ArrayTask();
+// Empty object for task
+export const objTask = {
+  description: '',
+  complete: '',
+};
+
+export function enableEdition(task) {
+  const { description, complete, index } = task;
+
+  // Upload content to the object
+  objTask.description = description;
+  objTask.complete = complete;
+  objTask.index = index;
+
+  // Fill the input
+  taskInput.value = description;
+
+  edit = true;
+}
+
+export function eraseTask(e, index, task) {
+  if (e.target.classList.contains(`specific-${index}`)) {
+    enableEdition(task);
+    listofTasks.saveLocalStorage();
+  }
+}
```

Comment:
You have declared your functions with 3 arguments. But later you have called the functions with an extra argument. 
https://github.com/yersel500/to-do-list-review/blob/28a7bf213ccadb80f6b3312a08cdcc6bcbe74235/src/modules/ui.js#L54
So I think you can remove the last argument which is redundant.

Reply:
Hi @Behnam1369  the comments were solved. Please help me with the final review